rcl/test/rcl/test_subscription.cpp

```diff
@@ -77,6 +77,29 @@ class CLASSNAME (TestSubscriptionFixture, RMW_IMPLEMENTATION) : public ::testing
   }
 };
 
+void
+wait_for_established_subscription(
+  const rcl_publisher_t * publisher,
+  size_t max_tries,
+  int64_t period_ms,
+  bool & success)
+{
+  size_t iteration = 0;
+  rcl_ret_t ret = RCL_RET_OK;
+  size_t subscription_count = 0;
+  do {
+    ++iteration;
+    ret = rcl_publisher_get_subscription_count(publisher, &subscription_count);
+    ASSERT_EQ(RCL_RET_OK, ret) << rcl_get_error_string().str;
+    if (subscription_count > 0) {
```

Comment:
subscription_count could be greater than 1? i think that we are expecting subscription_count == 1.